doc/ref_arch/kubernetes/chapters/chapter02.md

```diff
@@ -82,14 +82,15 @@ Reference to "Architecture" in this chapter refers to the NFVI Hardware (e.g. ph
 | `req.inf.ntw.16` | Network | The Architecture **should**  support co-exisence of multiple Conatiner Network Interface (CNI). |
 | `req.inf.acc.01` | Acceleration | The Architecture **should** support Application Specific Acceleration. |
 | `req.inf.acc.02` | Acceleration | The Architecture **should** support NFVI Acceleration (such as SmartNICs). |
```

Comment:
I am not sure what NFVI implies here. The need to have underlying RA1 in order to use SmartNICs? If that is the case I do not agree, I think one of the benefits of SmartNICs can be the omission of any sort of IAAS while still maintaining network automation. 

If that is not implied then I would just say "The Architecture **should** support acceleration and/or network automation via SmartNICs".